smartenergy/ml_service/models/agents/no_trading_agent.py

```diff
@@ -0,0 +1,41 @@
+from numpy import argmax, zeros
+from .base import Agent
+
+
+class NoTrainingAgent(Agent):
+
+    def __init__(self, action_space, hyperparameters={}):
+        self.action_space = action_space
+        self.hyperparameters = hyperparameters
+        self.last_state = None
+        self.last_actions = None
+        self.last_expected_rewards = None
+
+    # should we move to this to a new class called base_heuristics?
+    def train(self, memories):
+        pass
+
+    def get_action(self, state, random):
+        actions = {}
```

Comment:
Note that all get_action, get_state_value and get_expected_reward methods should return a dictionary that has the same structure as the self.action_space object. That way those values can be mapped to the specific connections accordingly. In this specific case, the get_state_value and get_action methods are the same: they both return dictionaries full with zeroes. So you can simply reference one from the other one (see code).
```suggestion
        actions = self.get_state_value(state)
```